specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -5279,6 +5644,74 @@
         }
       },
       "description": "ExpressRouteConnection list."
+    },
+    "RoutingConfiguration": {
+      "description": "Routing Configuration indicating the associated and propagated route tables for this connection.",
+      "properties": {
+        "associatedRouteTable": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "The resource id RouteTable associated with this RoutingConfiguration."
+        },
+        "propagatedRouteTables": {
+          "$ref": "#/definitions/PropagatedRouteTable",
+          "description": "The list of RouteTables to advertise the routes to."
+        },
+        "vnetRoutes": {
+          "$ref": "#/definitions/VnetRoute",
+          "description": "List of routes that control routing from VirtualHub into a virtual network connection."
+        }
+      }
+    },
+    "PropagatedRouteTable": {
+      "description": "The list of RouteTables to advertise the routes to.",
+      "properties": {
+        "labels": {
+          "type": "array",
+          "description": "The list of labels.",
+          "items": {
+            "type": "string"
+          }
+        },
```

Comment:
I'm not very familiar with routeTables.  What is the overlap between labels and the resource IDs?  It seems like each routeTable has 1..N labels but they are at the entire routeTable level, not per route.  So what does labels end up doing here?

Reply:
This is the type of info that is ideally contained in property descriptions.

---
In reply to: [417594558](https://github.com/Azure/azure-rest-api-specs/pull/9206#discussion_r417594558) [](ancestors = 417594558)